README.md

```diff
@@ -24,6 +24,6 @@ Detailed docs are available [here](http://ij.bazel.build).
 
 ## Building the plugin
 
-Install Bazel, then run `bazel build //ijwb:ijwb_bazel --define=ij_product=intellij-latest`
+Install Bazel, then run `bazel build //ijwb:ijwb_bazel_zip --define=ij_product=intellij-latest`
 from the project root. This will create a plugin jar in
```

Comment:
Thanks for the patch! Change the 'jar' reference here to 'zip' as well?

Reply:
Good catch! Fixed.